docs/2-advanced/07-accessibility.md

```diff
@@ -198,127 +170,134 @@ Will result in the shadow DOM as:
 ```
 
 The `accessibilityAttributes` property is currently supported in:
+* [Avatar](https://sap.github.io/ui5-webcomponents/nightly/components/Avatar/)
```

Comment:
Not sure if it's correct to point nightly documentation.

Reply:
All links have been pointing to nightly since the creation of the documentation. Should I change all of them?

Reply:
Let's keep it this way for now and we will handle this. In general it should not point to nightly and ideally be relative URL.